rust+wasm/cargo-generate.toml

```
@@ -73,7 +58,42 @@ type = "bool"
 prompt = "Set up Rust benchmarking?"
 default = false
 
-[conditional.'crate_type == "bin"'.placeholders.docker]
+[conditional.'within_cargo_project == true && is_init == true'.placeholders.wasm_project]
+type = "bool"
+prompt = "Want to set up a Wasm package/workspace of the project?"
```

```suggestion
prompt = "Want to set up a Wasm package/workspace in the project?"
```

Not sure, but maybe "in" works better here?